.travis.yml

```diff
@@ -1,70 +0,0 @@
-language: cpp
```

Comment:
Did you mean to delete this?

Reply:
See comment above, trying something in Appveyor CI